apps/tsconfig.json

```diff
@@ -36,7 +36,11 @@
     "allowJs": true,
     "baseUrl": "./",
     "paths": {
-      "@cdo/apps/*": ["./src/*"]
+      "@cdo/apps/*": ["./src/*"],
+      "@cdo/music/locale": ["./src/music/locale-do-not-import.js"],
+      "@cdo/aichat/locale": ["./src/aichat/locale-do-not-import.js"],
+      "@cdo/standaloneVideo/locale": ["./src/standaloneVideo/locale-do-not-import.js"],
+      "@cdo/locale": ["./src/util/locale-do-not-import.js"],
```

Comment:
🤔 `ts-loader` doesn't pass the webpack config aliases in. We basically avoided this by using `require()` calls (not really valid in TS / ESM syntax, but not flagged by babel) that were not evaluated by TS, and fell back to webpack for resolution. Switching to `esbuild` and/or using new style imports raises this question, because I don't see an easy way to have a separate tsconfig for dev vs prod vs test. I'm guessing there's a way.... but its also ugly to duplicate our locale aliases. Open question, maybe for @sanchitmalhotra126 ?

Reply:
Hmm, I'm not sure either...couple things/questions -
- So we are currently `import`ing many JS files in TS by specifically including files/paths in the `"include"` list above; once listed here, it seems like we can `import` these just fine in TS ([PR](https://github.com/code-dot-org/code-dot-org/pull/51976)). We did go with this tradeoff of manually specifying file paths though, I believe for the sake of build time. I wonder if with what you've been looking into, we could possibly try including all JS files here?
- Also I believe the `"@cdo/apps/*"` alias has been working fine with modern imports in TS files. I'm not sure why this path resolves but the locale file paths don't. I wonder if we need to manually include these in the `include` list above...? They should be captured by the file paths already though (e.g. `apps/src/music/**/*.js`)

Reply:
Found a resolution to the build time: run typechecking in parallel with `ts-loader`: https://github.com/code-dot-org/code-dot-org/pull/53778 . This lets us simplify include to be "all JS and TS" without impacting build performance.